Libraries/LibWebView/URL.cpp

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ * Copyright (c) 2023, 2025 Tim Flynn <trflynn89@serenityos.org>
```

Comment:
FYI we separate years with a hyphen (I thought we had a linter for this, but I guess not).
```c++
* Copyright (c) 2023-2025 Tim Flynn <trflynn89@serenityos.org>
```

(You also don't need to include me in this change, no worries)